LOAN/LoansGeneral/LoansGeneral.rdf

```diff
@@ -482,7 +492,7 @@
 		</rdfs:subClassOf>
 		<rdfs:label>payment history</rdfs:label>
 		<skos:definition>record of actual payments of principal, interest, and other related amounts made by a borrower to a lender or servicer in order to fulfill their re-payment obligation</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote>Similar to a PaymentSchedule except that the dates and amounts are actuals.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote>Payment history reflects a subset of the events that might be included in the overall explicit contract event schedule for a credit agreement, which would also include rate reset events, for example.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
how is the subset selected?
how are missed/late payments recorded?
or early/over payments?